config/locales/en.yml

```diff
@@ -33,6 +33,58 @@ en:
     service:
       ok: 'OK'
       down: 'Down'
+  hyku:
```

Comment:
Can you add the corresponding `es` translations.  Using google translate would be good enough for a first pass